packages/contracts-bedrock/contracts/test/MintManager.t.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { CommonTest } from "./CommonTest.t.sol";
+import { MintManager } from "../governance/MintManager.sol";
+import { GovernanceToken } from "../governance/GovernanceToken.sol";
+
+contract MintManager_Test is CommonTest {
+    address constant owner = address(0x1234);
+    address constant rando = address(0x5678);
+    GovernanceToken internal gov;
+    MintManager internal manager;
+
+    function setUp() external {
+        vm.prank(owner);
+        gov = new GovernanceToken();
+
+        vm.prank(owner);
+        manager = new MintManager(owner, address(gov));
+
+        vm.prank(owner);
+        gov.transferOwnership(address(manager));
+    }
+
+    function test_constructor_succeeds() external {
+        assertEq(manager.owner(), owner);
+        assertEq(address(manager.governanceToken()), address(gov));
+    }
+
+    function test_mint_fromOwner_succeeds() external {
+        // Mint once.
+        vm.prank(owner);
```

Comment:
I got this suggestion recently and think it's easier to reason about/maintain.

```suggestion
        vm.prank(manager.owner());
```

Reply:
This creates problems with `vm.prank` because the call to `manager.owner()` gets pranked e.g. on line 57